templates/search_results.hbs

```diff
@@ -1,9 +1,39 @@
 <div class="container-divider"></div>
 <div class="container">
-    <nav class="sub-nav custom-page-search-mini">
-      {{breadcrumbs}}
-      {{search scoped=true submit=false}}
-    </nav>
+  <nav class="sub-nav custom-page-search-mini">
+    {{breadcrumbs}}
+    {{search scoped=true submit=false}} 
+    <script>
+      $( document ).ready(function() {  
+        function getParam(n) {
+          var v = window.location.search.substring(1).split("&");
+
+          for (var i = 0; i < v.length; i++) {
+            var p = v[i].split("=");
+            if (p[0] == n) {
+              return decodeURIComponent(p[1]);
+            }
+          }
+
+          return "";
+        }
+
+        var categoryId = getParam('category');   
+        if (categoryId != "") {
+          var firstBreadcrumb = $(".custom-page-search-mini > .breadcrumbs > li:nth-child(1)");
+          
+          var presetCategories = {
+            "360001452494": "CET", 
+            "360003927054": "CET Developer",
+            "360001609774": "Catalogues",
+            "360001609754": "MyConfigura"
+          };
+          
+          firstBreadcrumb.after("<li title=\"CET\"><a href=\"/hc/en-us/categories/"+categoryId+"\">"+presetCategories[categoryId]+"</a></li>");
```

Comment:
It's safer to add a check if the `presetCategories[categoryId]` is not undefined. In case we add new categories in the future.